float64 lon
float64 lat
float64 groundSpeed
time measurementTime